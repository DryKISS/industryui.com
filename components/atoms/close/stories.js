/**
 * CLose
 */

// React
import React from 'react'

// Storybook
import { storiesOf } from '@storybook/react'

// UI
import { Close } from './'
import Readme from './README.md'

const CenterDecorator = storyFn =>
  <div style={{
    float: 'left',
    margin: '1rem 1rem 1rem .5rem'
  }}
  >
    {storyFn()}
  </div>

storiesOf('Atoms/Close', module)

  .addDecorator(CenterDecorator)

  .addParameters({
    readme: {
      sidebar: Readme
    }
  })

  .add('Default', () =>
    <Close click={() => {}} />
  )
