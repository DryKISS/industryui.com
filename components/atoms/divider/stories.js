/**
 * Divider
 */

// React
import React from 'react'

// Storybook
import { storiesOf } from '@storybook/react'

// UI
import { Divider } from './'
import Readme from './README.md'

storiesOf('Atoms/Divider', module)

  .addParameters({
    readme: {
      sidebar: Readme
    }
  })

  .add('Default', () =>
    <Divider />
  )

  .add('Medium', () =>
    <Divider size='md' />
  )

  .add('Large', () =>
    <Divider size='lg' />
  )

  .add('Custom Colour', () =>
    <Divider colour='red' size='lg' />
  )
