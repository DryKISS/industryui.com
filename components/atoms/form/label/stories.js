/**
 * Label
 */

// React
import React from 'react'

// Storybook
import { storiesOf } from '@storybook/react'

// UI
import { Label } from './'
import Readme from './README.md'

storiesOf('Atoms/Form/Label', module)

  .addParameters({
    readme: {
      sidebar: Readme
    }
  })

  .add('Default', () =>
    <Label children='' text='Select your gender' />
  )
