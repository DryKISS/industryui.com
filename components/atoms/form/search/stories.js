/**
 * Search
 */

// React
import React from 'react'

// Storybook
import { storiesOf } from '@storybook/react'
import { action } from '@storybook/addon-actions'

// UI
import { Search } from '../'
import Readme from './README.md'

storiesOf('Atoms/Form/Search', module)

  .addParameters({
    readme: {
      sidebar: Readme
    }
  })

  .add('Default', () =>
    <Search onSearch={action('query')} />
  )

  .add('With placeholder', () =>
    <Search placeholder='Company name' onSearch={action('query')} />
  )

  .add('With default value', () =>
    <Search value='XYZ' onSearch={action('query')} />
  )

  .add('With custom label', () =>
    <Search label='Go' onSearch={action('query')} />
  )
