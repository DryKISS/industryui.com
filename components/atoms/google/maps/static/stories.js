/**
 * Static Map
 */

// React
import React from 'react'

// UI
import { GoogleMapsStatic, GoogleMapsStaticPath } from './'
import Readme from './README.md'

export default {
  title: 'Atoms/Google/Static Map',
  component: GoogleMapsStatic,
  parameters: {
    readme: {
      sidebar: Readme
    }
  }
}

export const main = () => (
  <GoogleMapsStatic size='600x600' apiKey=''>
    <GoogleMapsStaticPath
      color='0xff0000ff'
      weight='5'
      points={[
        { lat: 40.737102, lng: -73.990318 },
        '40.749825,-73.987963',
        { lat: 40.752946, lng: -73.987384 },
        { lat: 40.755823, lng: -73.986397 }
      ]}
    />
  </GoogleMapsStatic>
)
