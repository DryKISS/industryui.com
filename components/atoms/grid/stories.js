/**
 * Grid
 */

// React
import React from 'react'

// Storybook
import { storiesOf } from '@storybook/react'

// UI
import { Container, Row, Column } from './'
import Readme from './README.md'

import { StyledColumn, colStyle } from './__mocks__/grid'

storiesOf('Atoms/Grid', module)

  .addParameters({
    readme: {
      sidebar: Readme
    }
  })

  .add('Default', () =>
    <Container>
      <Row>
        <Column>
          11
        </Column>
        <Column>
          12
        </Column>
      </Row>
      <Row>
        <Column>
          21
        </Column>
        <Column>
          22
        </Column>
        <Column>
          23
        </Column>
      </Row>
      <Row>
        <Column>
          31
        </Column>
      </Row>
    </Container>
  )

  .add('Size depending on screen width', () =>
    <Container>
      <Row>
        <StyledColumn md={3} lg={6} lx={9}>
          I love wide screen
        </StyledColumn>
        <StyledColumn md={9} lg={6} xl={3}>
          I love narrow screen
        </StyledColumn>
      </Row>
    </Container>
  )

  .add('With column sizes & offsets', () =>
    <Container>
      <Row>
        <StyledColumn md={4}>
          {'md=4'}
        </StyledColumn>
        <StyledColumn style={colStyle} md={4} offset={{ md: 4 }}>
          {'md=4 offset={md:4}'}
        </StyledColumn>
      </Row>
      <Row>
        <StyledColumn md={3} offset={{ md: 3 }}>
          {'md=3 offset={md:3}'}
        </StyledColumn>
        <StyledColumn md={3} offset={{ md: 3 }}>
          {'md=3 offset={md:3}'}
        </StyledColumn>
      </Row>
      <Row>
        <StyledColumn md={6} offset={{ md: 3 }}>
          {'md=6 offset={md:3}'}
        </StyledColumn>
      </Row>
    </Container>
  )
