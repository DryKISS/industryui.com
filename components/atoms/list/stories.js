/**
 * LIST
 */

// React
import React from 'react'

// Storybook
import { storiesOf } from '@storybook/react'
import { withKnobs, boolean, object } from '@storybook/addon-knobs'

import { List, ListItem } from '../../'
import Readme from './README.md'

storiesOf('Atoms/List', module)

  .addDecorator(withKnobs)
  .addParameters({
    readme: {
      sidebar: Readme
    }
  })

  .add('Default', () =>
    <List>
      <ListItem>Item1</ListItem>
      <ListItem>Item2</ListItem>
      <ListItem>Item3</ListItem>
    </List>
  )

  .add('Group', () =>
    <List group unstyled>
      <ListItem>Item1</ListItem>
      <ListItem>Item2</ListItem>
      <ListItem>Item3</ListItem>
    </List>
  )

  .add('Flush', () =>
    <List flush group unstyled>
      <ListItem>Item1</ListItem>
      <ListItem>Item2</ListItem>
      <ListItem>Item3</ListItem>
    </List>
  )

  .add('Knobbed', () =>
    <List
      flush={boolean('flush', false)}
      group={boolean('group', false)}
      unstyled={boolean('unstyled', false)}
      inline={boolean('inline', false)}
      style={object('Style', { color: 'blue', listStyleType: 'lower-greek' })}
    >
      <ListItem>Item1</ListItem>
      <ListItem>Item2</ListItem>
      <ListItem>Item3</ListItem>
    </List>
  )
