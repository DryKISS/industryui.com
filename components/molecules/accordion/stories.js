/**
 * Accordion
 */

import React from 'react'

import { storiesOf } from '@storybook/react'

import { Accordion, AccordionItem } from '../../'

import Readme from './README.md'

storiesOf('Molecules/Accordion', module)
  .addParameters({
    readme: {
      sidebar: Readme
    }
  })

  .add('Default', () => (
    <Accordion>
      <AccordionItem title="Item 1">
        Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon
        officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf
        moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim
        keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur
        butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably
        haven't heard of them accusamus labore sustainable VHS.
      </AccordionItem>
      <AccordionItem title="Item 2">
        Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon
        officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf
        moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim
        keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur
        butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably
        haven't heard of them accusamus labore sustainable VHS.
      </AccordionItem>
      <AccordionItem title="Item 3">
        Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon
        officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf
        moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim
        keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur
        butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably
        haven't heard of them accusamus labore sustainable VHS.
      </AccordionItem>
    </Accordion>
  ))

  .add('Open', () => (
    <Accordion>
      <AccordionItem open={true} title="Item 1">
        Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon
        officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf
        moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim
        keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur
        butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably
        haven't heard of them accusamus labore sustainable VHS.
      </AccordionItem>
      <AccordionItem title="Item 2">
        Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon
        officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf
        moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim
        keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur
        butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably
        haven't heard of them accusamus labore sustainable VHS.
      </AccordionItem>
      <AccordionItem title="Item 3">
        Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon
        officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf
        moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim
        keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur
        butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably
        haven't heard of them accusamus labore sustainable VHS.
      </AccordionItem>
    </Accordion>
  ))
