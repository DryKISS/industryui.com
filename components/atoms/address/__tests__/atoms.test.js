import { defaultStory } from './stories'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(defaultStory())
    .toJSON()
  expect(tree).toMatchSnapshot()
})