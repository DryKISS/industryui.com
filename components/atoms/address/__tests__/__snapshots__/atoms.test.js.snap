// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  margin-bottom: 1rem;
}

<address
  className="c0"
>
  <div>
    24 Accacia Road
  </div>
  <div>
    Grandhouse
  </div>
  <div>
    Bermondsey
  </div>
  <div>
    London
  </div>
  <div>
    Sussex
  </div>
  <div>
    W1 2N
  </div>
  <div>
    United Kingdom
  </div>
</address>
`;
